build/bazelbuilder/bazeltest.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -xuo pipefail
+
+bazel test //pkg:small_tests //pkg:medium_tests
```

Comment:
What about large/enormous?

Reply:
I added them (there are 5 large tests and 0 enormous tests, so it doesn't make that much of a difference now).